src/encoder/encoderflacsettings.cpp

```diff
@@ -0,0 +1,122 @@
+/****************************************************************************
+                   encoder.cpp  - encoder API for mixxx
+                             -------------------
+    copyright            : (C) 2017 by Josep Maria Antolín
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "encoder/encoderflacsettings.h"
+#include "recording/defs_recording.h"
+
+const int EncoderFlacSettings::DEFAULT_QUALITY_VALUE = 5;
+const QString EncoderFlacSettings::BITS_GROUP = "FLAC_BITS";
+const QString EncoderFlacSettings::GROUP_COMPRESSION = "FLAC_COMPRESSION";
+
+ 
+EncoderFlacSettings::EncoderFlacSettings(UserSettingsPointer pConfig)
+{
+    m_pConfig = pConfig;
+
+    QList<QString> names;
+    names.append("16 bits");
+    names.append("24 bits");
+    m_radioList.append(OptionsGroup("Bits", BITS_GROUP, names));
```

Comment:
Yes, probably bit depth is a better description for the group box, but then I don't know how to make that translatable. I know we put "tr()" for that, but I'm not sure if i can use it here, or if it would be dificult to do so.